src/shared/install.h

```diff
@@ -117,69 +117,89 @@ struct UnitFileInstallInfo {
 bool unit_type_may_alias(UnitType type) _const_;
 bool unit_type_may_template(UnitType type) _const_;
 
+typedef int (*mac_callback)(const char *path, void *userdata);
```

Comment:
could you add a `_t` suffix to this typedef? We don't follow this strictly, but usually we define enum and struct typedefs in CamelCase and all other typedefs with the _t suffix, so please do so here too.

Reply:
done